tests/orquestra/integrations/qiskit/_get_provider_test.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import Mock, create_autospec
+
+import pytest
+from qiskit_ibm_provider import IBMAccountError  # type: ignore
+from qiskit_ibm_provider.api.exceptions import RequestsApiError  # type: ignore
+
+from orquestra.integrations.qiskit import _get_provider
+
+
+@pytest.fixture
+def hub():
+    return "ibm-q"
+
+
+@pytest.fixture
+def group():
+    return "open"
+
+
+@pytest.fixture
+def project():
+    return "main"
+
+
+@pytest.fixture
+def mock_ibmprovider(monkeypatch: pytest.MonkeyPatch):
```

Comment:
praise: woah! this monkeypatch stuff seems super useful! Will definitely take a look for myself.

Reply:
Mocking can get weird, happy to show you how we use it in other repos